core/src/main/java/org/apache/carbondata/core/metadata/schema/table/column/ColumnSchema.java

```diff
@@ -87,13 +88,6 @@
    */
   private int columnGroupId = -1;
 
-  /**
-   * Used when this column contains decimal data.
-   */
-  private int scale;
-
-  private int precision;
```

Comment:
I think there would be an issue while reading gson if we remove these fields. we will lose the data of it if we remove them. Better keep them for backward compatibility.

Reply:
fixed